lib/tasks/play.js

```diff
@@ -66,8 +67,20 @@ class TaskPlay extends Task {
         }
       }
     } catch (err) {
-      if (timeout) clearTimeout(timeout);
       this.logger.info(err, `TaskPlay:exec - error playing ${this.url}`);
+      this.playComplete = true;
+      if (err.message === 'File not found') {
```

Comment:
The string 'File not found' should be 'File Not Found' as returned from drachtio-msrf change https://github.com/drachtio/drachtio-fsmrf/blob/v3.0.46/lib/endpoint.js#L345

Without this, the fix is not working

Reply:
fixed in https://github.com/jambonz/jambonz-feature-server/pull/990/commits/d30f00f369763a554d0d0ba50d342024231da0f1